src/scripts/accesskit/disable_gifs.js

```diff
@@ -67,13 +84,20 @@ export const main = async function () {
     ${keyToCss('communityHeaderImage', 'bannerImage')}[style*=".gif"]
   `;
   pageModifications.register(gifBackgroundImage, processBackgroundGifs);
+
+  pageModifications.register(await keyToCss('rows'), processRows);
```

Comment:
`keyToCss()` is synchronous now so we can also drop the await

```suggestion
  pageModifications.register(`${postSelector} ${keyToCss('rows')}`, processRows);
```